src/__tests__/helper.test.js

```diff
@@ -0,0 +1,99 @@
+
+import { getContentHeight, getItemPosition, checkScreenFit } from '../helper';
+
+const MockLayoutMapWithArea = {
+  0: {y: 0,height: 600,isArea: undefined},
+  1: {y: 600,height: 150,isArea: true},
+  2: {y: 750,height: 500,isArea: undefined},
+};
+
+const MockLayoutMapWithoutArea = {
+  0: {y: 0, height: 600, isArea: undefined},
+  1: {y: 600, height: 45, isArea: undefined},
+  2: {y: 645, height: 20, isArea: undefined},
+  3: {y: 665, height: 45, isArea: undefined},
+  4: {y: 710, height: 20, isArea: undefined},
+  5: {y: 730, height: 40, isArea: undefined},
+  6: {y: 770, height: 500, isArea: undefined},
+}
+
+const MockFocusedField = 1;
+
+const MockSafeMarginContentHeight = 70;
+
+describe('Testing of Avoidable helper functions', () => {
```

Comment:
I don't think the word "testing" is necessary here, it is obvious that you are testing something. "Avoidable helper" is just enough, IMO.